api-audit/src/main/java/com/capitalone/dashboard/evaluator/CodeReviewEvaluatorLegacy.java

```diff
@@ -194,5 +195,29 @@ private CodeReviewAuditResponse getErrorResponse(CollectorItem repoItem, String
         return allPeerReviews;
     }
 
+    private void auditIncrementVersionTag(CodeReviewAuditResponse codeReviewAuditResponse, Commit commit, CodeReviewAuditStatus directCommitIncrementVersionTagStatus) {
+        if (CommonCodeReview.matchIncrementVersionTag(commit.getScmCommitLog(), settings)) {
+            codeReviewAuditResponse.addAuditStatus(directCommitIncrementVersionTagStatus);
+        }
+    }
+
+    private void auditServiceAccountChecks(CodeReviewAuditResponse codeReviewAuditResponse, Commit commit) {
+        if (isServiceAccount(commit.getScmAuthorLDAPDN()))  {
+           codeReviewAuditResponse.addAuditStatus(CodeReviewAuditStatus.COMMITAUTHOR_EQ_SERVICEACCOUNT);
+           // check for increment version tag and flag Direct commit by Service account
+            auditIncrementVersionTag(codeReviewAuditResponse, commit, CodeReviewAuditStatus.DIRECT_COMMIT_INCREMENT_VERSION_TAG_SERVICE_ACCOUNT);
+       }
+    }
+
+    private void auditNonServiceAccountChecks(CodeReviewAuditResponse codeReviewAuditResponse, Commit commit) {
+        if (!isServiceAccount(commit.getScmAuthorLDAPDN()))  {
+           auditIncrementVersionTag(codeReviewAuditResponse, commit, CodeReviewAuditStatus.DIRECT_COMMIT_INCREMENT_VERSION_TAG_NON_SERVICE_ACCOUNT);
+        }
+    }
+
+    private boolean isServiceAccount(String scmAuthorLDAPDN) {
+        return !StringUtils.isEmpty(scmAuthorLDAPDN) && CommonCodeReview.checkForServiceAccount(scmAuthorLDAPDN, settings);
```

Comment:
I think the checkForServiceAccount already checks for empty string. In which case, we don't need this method?

Reply:
@tabladrum  - done